tests/tests/test_timetable.py

```diff
@@ -98,16 +111,51 @@ def test_get_timetable(
     assert simulation_response.status_code == 201
     simulation = _SimulationResponse(**simulation_response.json())
     assert len(simulation.ids) == 2
-
     response = requests.get(f"{service_url}timetable/{timetable_id}/")
     timetable = _TimetableResponse(**response.json())
     assert timetable.id == small_scenario.timetable
     assert timetable.name == "timetable"
-    assert len(timetable.train_schedules) == 2
+    assert len(timetable.detailed_train_schedules) == 2
     expected_schedules = [
-        _TrainScheduleDetails(simulation.ids[0], "West to South East 1", 3600, west_to_south_east_path.id),
-        _TrainScheduleDetails(simulation.ids[1], "West to South East 3", 5100, west_to_south_east_path.id),
+        _TrainScheduleDetails(
+            simulation.ids[0],
+            "West to South East 1",
+            3600,
+            4384.388061579815,
+            west_to_south_east_path.id,
+            small_scenario.timetable,
+            fast_rolling_stock,
+            0.0,
+            ["new train", "west", "south east"],
+            [],
+            None,
+            "STANDARD",
+            None,
+            None,
+            {"base": 6041639237.0, "eco": None},
+            0,
+            45549.5653000392,
+        ),
```

Comment:
Some types don't match, it causes some warnings in pycharm.

The int/float and None are not that important, but the "expected `Iterable[Dict]`, got `Dict[...]` instead" is more worrying

Reply:
Those should be fixed now.